src/utils.test.js

```diff
@@ -161,73 +159,6 @@ describe('utils', () => {
     });
   });
 
-  describe('setInterval custom hook', () => {
-    it('should set an interval and update an outside value', () => {
-      // Arrange
-      jest.useFakeTimers();
-      let counter = 0;
-
-      // Act
-      renderHook(() =>
-        useInterval({
-          runOnStart: false,
-          delay: 100,
-          callback: () => {
-            counter++;
-          },
-        }),
-      );
-      jest.runOnlyPendingTimers();
-
-      // Assert
-      expect(counter).toBe(1);
-    });
-
-    it('should set an interval and pause when delay is null', () => {
-      // Arrange
-      jest.useFakeTimers();
-      let counter = 0;
-      // Act
-      const { rerender } = renderHook(
-        ({ delay }) =>
-          useInterval({
-            runOnStart: false,
-            delay: delay,
-            callback: () => {
-              counter++;
-            },
-          }),
-        { initialProps: { delay: 100 } },
-      );
-      // Assert
-      jest.advanceTimersByTime(600);
-      rerender({ delay: null });
-      jest.advanceTimersByTime(800);
-      expect(counter).toBe(6);
-    });
-
-    it('should set an interval and run callback before start', () => {
-      // Arrange
-      jest.useFakeTimers();
-      let counter = 0;
-      // Act
-      renderHook(() =>
-        useInterval({
-          runOnStart: true,
-          delay: 100,
-          callback: () => {
-            counter++;
-          },
-        }),
-      );
-      expect(counter).toBe(1);
-
-      jest.advanceTimersByTime(200);
-      // Assert
-      expect(counter).toBe(3);
-    });
-  });
-
```

Comment:
Is possible to keep the tests? If I am not wrong, the behavior should be the same as before, right?

Reply:
Yes, the behavior should be the same as before. 
But, now I changed it to use a different `useInterval` located in `src/hooks/useInterval/index.js`. 
Practically it is a bit like the previous one, but it is a little more flexible.

Reply:
> Practically it is a bit like the previous one, but it is a little more flexible.

But we loose tests :(

Reply:
I'm going to upload a commit testing this functionality in the `Shopify Component`